lib/ansible/module_utils/csharp/Ansible.Basic.cs

```diff
@@ -78,6 +78,7 @@ public class AnsibleModule
             { "tmpdir", "tmpdir" },
             { "verbosity", "Verbosity" },
             { "version", "AnsibleVersion" },
+            { "backup_file_name_template", "BackupFileNameTemplate" },
```

Comment:
Until we want to actually use this best to just ignore the value for now.

```suggestion
            { "backup_file_name_template", null },
```